extensions/Worker.Extensions.CosmosDB/src/Worker.Extensions.CosmosDB.csproj

```diff
@@ -7,7 +7,7 @@
 
     <!--Version information-->
     <VersionPrefix>4.1.0</VersionPrefix>
-    <VersionSuffix>-preview1</VersionSuffix>
+    <VersionSuffix>-preview2</VersionSuffix>
```

Comment:
Just realised we bumped this to 4.2.0

https://github.com/Azure/azure-functions-dotnet-worker/blob/0d49e191d454669fde6186559404df1e2ac1c107/extensions/Worker.Extensions.CosmosDB/src/Properties/AssemblyInfo.cs#LL7C81-L7C86

so this should be 4.2.0-preview1